tests/test_archives.py

```diff
@@ -168,31 +168,3 @@ def test_archive_delete(qapp, qtbot, mocker, borg_json_output):
     qtbot.waitUntil(lambda: 'Archive deleted.' in main.progressText.text(), **pytest._wait_defaults)
     assert ArchiveModel.select().count() == 1
     assert tab.archiveTable.rowCount() == 1
-
-
-def test_archive_rename(qapp, qtbot, mocker, borg_json_output):
```

Comment:
Will look at some docs online to add a new test here, according to the changes. Any help will also be appreciated!

Reply:
I tested using simple double mouse click event and it works

```
pos = tab.archiveTable.visualRect(tab.archiveTable.model().index(0, 4)).center()
 qtbot.mouseClick(tab.archiveTable.viewport(), QtCore.Qt.MouseButton.LeftButton, pos=pos)
 qtbot.mouseDClick(tab.archiveTable.viewport(), QtCore.Qt.MouseButton.LeftButton, pos=pos)
```

This will trigger the double click and rename action. You can then send new name and enter key.

Reply:
Thanks!